pkg/localcontroller/manager/incrementallearningjob.go

```diff
@@ -446,6 +464,41 @@ func newTrigger(t sednav1.Trigger) (trigger.Base, error) {
 	return trigger.NewTrigger(triggerMap)
 }
 
+func (im *IncrementalJobManager) getTrainOrEvalModel(job *IncrementalLearningJob, jobStage sednav1.ILJobStage) *ModelInfo {
+	jobConditions := job.Status.Conditions
+
+	// TODO: globalmanager.type changes to common.type for gm and lc
+	var models []globalmanager.Model
+
+	for i := len(jobConditions) - 1; i >= 0; i-- {
+		var cond globalmanager.IncrementalCondData
+		jobCond := jobConditions[i]
+		if jobCond.Stage == sednav1.ILJobTrain && jobCond.Type == sednav1.ILJobStageCondCompleted {
+			(&cond).Unmarshal([]byte(jobCond.Data))
+			if cond.Output == nil {
+				continue
+			}
+
+			models = cond.Output.Models
+
+			break
+		}
+	}
+
+	if models == nil && len(models) != 2 {
+		return nil
+	}
+
+	switch jobStage {
+	case sednav1.ILJobTrain:
+		return &ModelInfo{Format: models[1].Format, URL: models[1].URL}
```

Comment:
can you add some comment here? can't understand here

Reply:
fix it